native_client/deepspeech.cc

```diff
@@ -1,4 +1,7 @@
 #include <algorithm>
+#ifdef _MSC_VER
+  #define _USE_MATH_DEFINES
+#endif
 #include <cmath>
```

Comment:
@lissyx Why here `cmath`but in [c_speech_features.cpp](https://github.com/carlfm01/DeepSpeech/blob/d519b7559f526e50f7727d8b55dfa29278fa98ed/native_client/c_speech_features/c_speech_features.cpp#L5) used `math.h`?

Reply:
good question

Reply:
part of answer: https://github.com/carlfm01/DeepSpeech/commit/5a6afde0e1c1a7458ea89bd991f3875d8f172786